torch_xla/csrc/ops/ops_xla_shape_fn.cpp

```diff
@@ -0,0 +1,7 @@
+#include "torch_xla/csrc/ops/ops_xla_shape_fn.h"
+
+namespace torch_xla {
+
+xla::Shape AbsOutputShape(const XlaValue& input) { return input.xla_shape(); }
```

Comment:
i thought you already had some kind of shape functions you could just call for the 'xla shape' equivalent of shape functions?

Reply:
for complicated Ops we do have a `NodeOutputShape` function, but for easier ops like `abs` we know the output shape is just the input shape we used to pass the `xla::shape` back directly instead of defining a shape function.